### DISCUSSION TITLE
Is there anyway to download a small portion of mC4?

### DISCUSSION THREAD

**Discussion by sbmaruf** [google-research/text-to-text-transfer-transformer#835] [General]
Hi! Not sure if it is the right place to ask this question. Since the mT5 repo doesn't have any Discussion bar, I'm asking it here.

Is there any way we can download a small portion of mC4? More than 20T seems really large space. It would be nice if we can download the dataset partially (language-wise or even language-wise random samples).

**Comment by craffel** [comment]
Not really - unless I'm mistaken the only way to do this would be to manually modify the code so that it only processed one crawl, but that would still be very compute intensive (@adarob can confirm). FWIW there is a rumor that AI2 will release mC4 like they did for C4. Maybe @dodgejesse can comment.

**Reply by dodgejesse** [reply]
Yes, we're working on releasing mC4 in the next few weeks. We plan to allow downloading each of the languages separately, and we're considering releasing a downsampled version that'll be smaller (for convenience, e.g. when prototyping models).

**Reply by sbmaruf** [reply]
Thanks a lot for the reply. Eagerly waiting for the dataset.
